packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -16,6 +20,23 @@ describe('Sayt Driver Plugin', () => {
       autocomplete: (query, config, callback) => null,
     };
     driver = new SaytDriverPlugin();
+    config = {
+      collection: 'backup'
+    };
+    query = {
```

Comment:
Shouldn't this just be a string? The tests below seem to assert that `autocomplete()` will receive a query object, which doesn't seem to be true. From the following definition:
```
sendSaytApiRequest(query: string, config: QueryTimeAutocompleteConfig): Promise<string[]> {
    return this.core.sayt.autocomplete(query, config).then(this.autocompleteCallback);
  }
```
it appears that `this.core.sayt.autocomplete()` will receive a string as its first argument, as indicated in the parameters for `sendSaytApiRequest()`.

Ultimately it doesn't matter for the tests if we're mocking out that function, but I think we should at least try to be accurate in terms of the basic shape of the data we use to query.